src/App.test.jsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import App from './App';
+
+describe('App', () => {
+  it('should navigate between a list view and a detail view', async () => {
+    render(
+      <MemoryRouter initialEntries={['/characters']}>
+        <App />
+      </MemoryRouter>
```

Comment:
Be sure to add in a detail page test that initializes our detail page using initialEntries (i.e. initialEntries = { ['/characters/1 ] }.